packages/sdk-react/src/core/context/MoniteI18nProvider.test.tsx

```diff
@@ -153,3 +171,142 @@ describe('MoniteI18nProvider DatePicker', () => {
     );
   });
 });
+
+describe('MoniteI18nProvider currencyNumberFormat', () => {
+  it('returns locale codes if specified', async () => {
```

Comment:
Would you mind to change all `it` into `test`?
There is no difference between them but the whole code uses `test` so from my point of view it would be better to follow the same rule everywhere

Reply:
Done